modules/openapi-generator/src/test/java/org/openapitools/codegen/DefaultCodegenTest.java

```diff
@@ -570,6 +570,23 @@ public void testNullableProperty() {
         Assert.assertTrue(property.isNullable);
     }
 
+    @Test
+    public void testAlias() {
+        final OpenAPI openAPI = TestUtils.parseSpec("src/test/resources/3_0/type_alias.yaml");
+        new InlineModelResolver().flatten(openAPI);
+
+        final DefaultCodegen codegen = new DefaultCodegen();
+        codegen.setOpenAPI(openAPI);
+
+        CodegenModel model = codegen.fromModel(
+                "MyParameterTextField",
+                openAPI.getComponents().getSchemas().get("MyParameterTextField")
+        );
+
+        Assert.assertTrue(model.isAlias);
+        Assert.assertEquals("string", model.dataType);
```

Comment:
In some of generators (e.g. Kotlin which supports Type Alias), it may expected that the `dataType` property has a specific model type for better type alias support.

ref https://github.com/OpenAPITools/openapi-generator/issues/2574